tests/test_features/test_pca.py

```diff
@@ -64,39 +65,101 @@ def binary(request):
     request.cls.dataset = Dataset(X, y)
 
 
+@pytest.fixture(scope="class")
+def continuous(request):
+    """
+    Creates a random regressor fixture.
+    """
+    X, y = make_regression(
+        n_samples=500, n_features=22, n_informative=8, random_state=2019
+    )
+
+    # Set a class attribute for continuous data
+    request.cls.continuous = Dataset(X, y)
+
+
 ##########################################################################
 ##PCA Tests
 ##########################################################################
 
 
-@pytest.mark.usefixtures("binary")
+@pytest.mark.usefixtures("binary", "continuous")
 class TestPCADecomposition(VisualTestCase):
     """
     Test the PCADecomposition visualizer
     """
 
-    @pytest.mark.xfail(
-        sys.platform == "win32", reason="images not close on windows (RMSE=4)"
-    )
+    def test_single(self):
+        """
+        Test single target.
+        """
+        visualizer = PCADecomposition(random_state=1998)
+        visualizer.fit(self.continuous.X)
+        visualizer.transform(self.continuous.X)
+        self.assert_images_similar(visualizer)
```

Comment:
Potentially this test would benefit from some `assert not hasattr` statements similar to continuous and discrete.

Reply:
I will make sure to add these!